test/js/.eslintrc

```diff
@@ -11,7 +11,45 @@
     "expect": true
   },
   "rules": {
-    "prefer-arrow-callback": 0,
-    "max-nested-callbacks": 0
+    "prefer-arrow-callback": 2,
+    "max-nested-callbacks": 0,
+    "no-empty": 0,
+    "no-console": 0,
+    "no-unused-vars": [2, { "varsIgnorePattern": "^h|React$" }],
+    "no-cond-assign": 1,
+    "space-before-blocks": 2,
+    "semi": 0,
+    "camelcase": 0,
+    "comma-style": 2,
+    "comma-dangle": [2, "only-multiline"],
+    "indent": [2, 2, {"SwitchCase": 1, MemberExpression: 0}],
+    "no-mixed-spaces-and-tabs": [2, "smart-tabs"],
+    "no-trailing-spaces": [2, { "skipBlankLines": true }],
+    "no-eval": 2,
+    "no-implied-eval": 2,
+    "no-new-func": 2,
+    "guard-for-in": 2,
+    "eqeqeq": 1,
+    "no-else-return": 2,
+    "no-redeclare": 2,
+    "no-dupe-keys": 2,
+    "radix": 2,
+    "strict": [2, "never"],
+    "no-shadow": 0,
+    "no-delete-var": 2,
+    "no-undef-init": 2,
+    "no-shadow-restricted-names": 2,
+    "handle-callback-err": 0,
+    "no-lonely-if": 2,
+    "keyword-spacing": 1,
+    "constructor-super": 2,
+    "no-this-before-super": 2,
+    "no-dupe-class-members": 2,
+    "no-const-assign": 2,
+    "prefer-spread": 2,
+    "no-useless-concat": 2,
+    "no-var": 2,
+    "object-shorthand": 2,
+    "react/jsx-uses-vars": 1
```

Comment:
I'm curious, but did extending the main `.eslintrc` not work? Because this feels like an extra config to maintain for any change in the main one aside from some (intentional?) rule differences from the main lint config.

Reply:
That's a good point. For some reason some rules were not inherited. I will make sure there is no duplication